### ISSUE TITLE
Add text clarifying offline usage of the ToIP stack

### ISSUE THREAD

**Issue by andorsk** [trustoverip/TechArch#36]
One of the things that came up in the TATF meeting October 7, is that it would be good to clarify where documentation ( if it exists ) on the offline trust layer. 

Will update this issue later with more details.

**Comment by andorsk** [comment]
Some comments from the meeting: 

>  Mobile Drivers License - dealing with offline mode - how are they doing it
>  - they have to download the keys (periodically) to continue for the reader
> - how to deal with not having access to a data registry
>  - has been done with PKD - has not yet been talked about.

**Comment by talltree** [comment]
I agree it is important for us to address the general category of offline use cases and to make some clear statements about how they can be addressed by the ToIP stack. 

_Proposed tags: `needs-PR` and `priority-2`_

**Comment by jospencer-460** [comment]
We're talking about a Presentation Type of Offline.  This doesn't mean that this needs to be reflected at the Use Case level.

**Comment by andorsk** [comment]
@jospencer-460 I seemed to have assigned you but I don't remember why "doh". My guess it was mentioned on the October 13 call. Could you confirm?

**Comment by jospencer-460** [comment]
I'm happy to be part of this discussion / decision.  I commented on this issue and we were using my comment to register me into the process.

**Comment by allant0** [comment]
Suggest we consider identifying within the layers that already exist in the architecture which ones support online & offline activities. To me it seems strange that only one layer is focused on offline access. All parts of the system need to handle what happens when they go "offline". It could be as simple as "report error that this functionality is not possible while not connected to the internet" or it could be more "support X, Y, Z activities for the entity locally and mandate synchronization of the state of X, Y upon resumption of online access".

**Comment by andorsk** [comment]
I'm gonna create this into a discussion for now, and then we can bring it back to issues once it's been scoped a little better. 

I can say that this recently came up in a meeting I was in, specifically about offline trust. It would be great if one day there is some clear documentation on this.

**Comment by andorsk** [comment]
As far as suggestions here: @jospencer-460 I'd love to work with you on it as well. There's a few areas that specifically interest me: 

1. Trust registries and how you build trust in an offline context.
2. Some constraints, I would imagine with some overlap with a CRDT.
